In [1]:
import numpy as np 
import pandas as pd
import statsmodels.formula.api as smf
import seaborn as sns

In [2]:
df = pd.read_csv('https://raw.githubusercontent.com/artamonoff/Econometrica/master/python-notebooks/data-csv/sleep75.csv')
df.shape

(706, 34)

In [3]:
fitted_plane = smf.ols(formula='sleep~totwrk+age', data=df).fit()
fitted_plane.params.round(2)

Intercept    3469.20
totwrk         -0.15
age             2.92
dtype: float64

## Ответ: Intercept = 3469.20; totwrk = -0.15; age = 2.92 (параметры оптимальной плоскости sleep на totwrk, age)